tests/test_abstract_repr.py

```diff
@@ -490,18 +490,18 @@ def test_paramobj_serialization(self, sequence):
         )
 
         s = json.dumps(
-            Pulse.ConstantDetuning(wf, 0.0, var, post_phase_shift=1.0),
+            Pulse.ConstantDetuning(wf, 0.0, var),
```

Comment:
The idea is first to test without the post_phase_shift and then with a post_phase_shift of 1 ?

Reply:
It was just to cover multiple cases: 
1. the value is not given 
2.the value is given as an arg 
3. the value is given as a kwarg